archiver.go

```diff
@@ -117,6 +117,10 @@ func (a *Archiver) do(j *Job) error {
 
 	if err := fetchAll(gr); err != nil {
 		var finalErr error
+		if err == git.NoErrAlreadyUpToDate {
```

Comment:
Add here a comment to describe the corner case that is handling this if: when you are updating a repository and all the references have been deleted, then a NoErrAlreadyUpToDate error will thrown. The references should be removed from the database, because now this repository has no references.